actionpack/lib/action_controller/metal/http_authentication.rb

```diff
@@ -506,7 +506,7 @@ def rewrite_param_values(array_params)
       # pairs by the standardized <tt>:</tt>, <tt>;</tt>, or <tt>\t</tt>
       # delimiters defined in +AUTHN_PAIR_DELIMITERS+.
       def raw_params(auth)
-        _raw_params = auth.sub(TOKEN_REGEX, "").split(/\s*#{AUTHN_PAIR_DELIMITERS}\s*/)
+        _raw_params = auth.sub(TOKEN_REGEX, "").split(/\s*#{AUTHN_PAIR_DELIMITERS}\s*/o)
```

Comment:
`/o` is a really obscure feature of Ruby regexps, and is likely to trip people out. I'd rather extract this in a constant for clarity.